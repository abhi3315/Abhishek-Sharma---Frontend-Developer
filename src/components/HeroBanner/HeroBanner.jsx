import RocketImage from '../../assets/rocket.webp';

function HeroBanner() {
	return (
		<div className='grid max-w-screen-xl px-4 md:px-10 py-4 mx-auto lg:gap-8 xl:gap-0 lg:py-16 lg:grid-cols-12 mt-10 '>
			<div className='mr-auto place-self-center lg:col-span-7'>
				<h1 className='max-w-2xl mb-4 text-4xl font-extrabold tracking-tight leading-none md:text-5xl xl:text-6xl dark:text-white'>
					SpaceX Shuttle Diary
				</h1>
				<p className='max-w-2xl mb-6 font-light text-gray-500 lg:mb-8 md:text-lg lg:text-xl dark:text-gray-400'>
					Stay updated with SpaceX rocket information for the latest news and updates on their
					innovative technology and space exploration missions.
				</p>
			</div>
			<div className='hidden lg:mt-0 lg:col-span-5 lg:flex'>
				<img src={RocketImage} alt='Rocket' />
			</div>
		</div>
	);
}

export default HeroBanner;
