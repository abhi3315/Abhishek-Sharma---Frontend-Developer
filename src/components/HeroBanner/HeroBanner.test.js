import { render, cleanup } from '@testing-library/react';

import HeroBanner from './HeroBanner';

afterEach(cleanup);

describe('HeroBanner', () => {
	it('should render correctly', () => {
		const { container } = render(<HeroBanner />);
		expect(container).toMatchSnapshot();
	});
});
