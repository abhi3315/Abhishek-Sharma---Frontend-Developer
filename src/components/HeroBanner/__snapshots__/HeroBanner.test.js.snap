// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroBanner should render correctly 1`] = `
<div>
  <div
    class="grid max-w-screen-xl px-4 md:px-10 py-4 mx-auto lg:gap-8 xl:gap-0 lg:py-16 lg:grid-cols-12 mt-10 "
  >
    <div
      class="mr-auto place-self-center lg:col-span-7"
    >
      <h1
        class="max-w-2xl mb-4 text-4xl font-extrabold tracking-tight leading-none md:text-5xl xl:text-6xl dark:text-white"
      >
        SpaceX Shuttle Diary
      </h1>
      <p
        class="max-w-2xl mb-6 font-light text-gray-500 lg:mb-8 md:text-lg lg:text-xl dark:text-gray-400"
      >
        Stay updated with SpaceX rocket information for the latest news and updates on their innovative technology and space exploration missions.
      </p>
    </div>
    <div
      class="hidden lg:mt-0 lg:col-span-5 lg:flex"
    >
      <img
        alt="Rocket"
        src="rocket.webp"
      />
    </div>
  </div>
</div>
`;
